examples/deprecated/centos-7.yaml

```diff
@@ -0,0 +1,24 @@
+# This example requires Lima v0.10.0 or later.
+# WARNING: RHEL-like below version 8 is unsupported, use at your own risk!
+images:
+- location: "https://cloud.centos.org/altarch/7/images/CentOS-7-x86_64-GenericCloud-2009.qcow2"
```

Comment:
Was there any specific reason to avoid 2111?
https://cloud.centos.org/altarch/7/images/CentOS-7-x86_64-GenericCloud-2111.qcow2

Reply:
Actually no. The only reason I didn't put was I thought all images should have a checksum to compare against and that image doesn't have on https://cloud.centos.org/altarch/7/images/sha256sum.txt. Also, there is no centos-7.2111 ISO. My conclusion was the image 2111 was an error.

I just test it and it worked. In fact, the build 2111 is associate with CentOS 8, as you can see on https://wiki.centos.org/Manuals/ReleaseNotes/CentOS8.2111, but it is Centos 7 as I could confirm.

Reply:
Thanks, I agree it might be an error 👀